Add missing standards, organize standards, and update links in source README (#48)
FuelLabs/sway-standards: README.md

## Standards

### Native Assets

> - [SRC-20; Token Standard](./standards/src20-token/) defines the implementation of a standard API for [Native Assets](https://docs.fuel.network/docs/sway/blockchain-development/native_assets) using the Sway Language.

Reviewer: Would be nice to have these in numerical order, will be helpful as the lists continue to grow

Author: I agree but in this case I do feel the token standard should go before any of the token extensions. We originally named it SRC-20 in homage to ERC-20 but in hindsight that might cause problems

Example of the SRC-5 implementation where a contract has an owner set at compile time.

#### SRC-6; Vault Standard Examples

##### [Multi Token Vault](./examples/src6-vault/multi_token_vault/)

> A barebones implementation of the vault standard that supports any number of sub vaults being created for every AssetId.

Reviewer: ```suggestion
A basic implementation of the vault standard that supports any number of sub vaults being created for every AssetId.
```

##### [Single Token Vault](./examples/src6-vault/single_token_vault/)

> A basic implemenation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.

Reviewer 2: ```suggestion
A basic implementation of the vault standard demonstrating how to restrict deposits and withdrawals to a single AssetId.
```

> A barebones implemenation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.

Reviewer: ```suggestion
A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId.
```

##### [Single Token Single Sub Vault](./examples/src6-vault/single_token_single_sub_vault/)

> A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.

Reviewer 2: ```suggestion
A basic implementation of the vault standard demonstrating how to restrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```

> A barebones implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.

Reviewer: ```suggestion
A basic implementation of the vault standard demonstrating how to constrict deposits and withdrawals to a single AssetId, and to a single Sub vault.
```
